src/config/args.rs

```diff
@@ -76,8 +78,9 @@ impl Args {
                 // We ignore this flag for now.
             } else if let Some(format) = parse_value("--format", &arg, &mut iter)? {
                 self.format = match &*format {
-                    "terse" => Format::Terse,
+                    "libtest-json" => Format::LibtestJSON,
```

Comment:
hmm why not just `json`?

Reply:
This was intentional, in case you would like to add "full" JSON support at some point.

Note that some of the events generated by the testing framework are not emitted, hence I was apprehensive about calling this "JSON". Up to you really.

Reply:
I have few qualms changing the json output to only mostly match libtest-json and to have more data inside that libtest doesn't emit ;) Let's just use `json`

Reply:
OK, `json` it is then!

Reply:
Done.